### PR TITLE
Delete Vaccine Schedule Section

### PR DESCRIPTION
**Pull request by naufalihsank**
### Changes
1. Delete vaccine schedule section in vaccine page
2. Setup airtable api
### Task
1. [[Web S10] [Vaksinasi] Perubahan Section di halaman Vaksinasi](https://trello.com/c/shjdwuWD/1019-web-s10-vaksinasi-perubahan-section-di-halaman-vaksinasi)
2. [[Web S10] [Vaksinasi -> Jadwal Vaksinasi] Pembuatan Halaman Jadwal Vaksinasi menggunakan data Airtable](https://trello.com/c/LhoCRCfm/969-web-s10-vaksinasi-jadwal-vaksinasi-pembuatan-halaman-jadwal-vaksinasi-menggunakan-data-airtable)
### Evidence
title: Delete Vaccine Schedule Section
project: Pikobar Website
participants: @naufalihsank @maulanayuseph @Ibwedagama @bangunbagustapa

### CODE REVIEW

File: api/vaksin.js [hunk 1]
```diff
@@ -25,17 +26,22 @@ export async function get () {
...
-        return {
-          id: doc.id,
-          ...doc.data()
-        }
-      }
+export async function getSchedule (params) {
+  try {
+    const baseURL = process.env.NUXT_ENV_AIRTABLE_URL
+    const headers = { Authorization: `Bearer ${process.env.NUXT_ENV_AIRTABLE_API_KEY}` }
+    const url = '/Informasi Vaksinasi Pikobar'
```
Comment (Ibwedagama) [inline api/vaksin.js:33]: hi @naufalihsank , I just wondering, is this `url` variable correct? or it's supposed to be `/informasi-vaksinasi-pikobar` ?

Comment (naufalihsank) [reply api/vaksin.js:33]: @Ibwedagama the airtable endpoint is `/Informasi%20Vaksinasi%20Pikobar`, so that `url` will be generated to the right endpoint

Comment (bangunbagustapa) [reply api/vaksin.js:33]: I'm curious too about that line

Comment (Ibwedagama) [reply api/vaksin.js:33]: Oh, I see, if that's the case then it's already the correct URL @naufalihsank  👍🏼

**Review by Ibwedagama — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

**Review by bangunbagustapa — APPROVED** [review]

**Pull request merged**